app/models/accounting/quickbooks/updater.rb

```diff
@@ -102,7 +102,7 @@ def extract_qb_data(txn)
 
       def changes
         unless last_updated_at && last_updated_at > max_updated_at
-          raise FullSyncRequiredError, "Last update was more than 30 days ago, please do a full sync"
+          raise FullSyncRequiredError, "Last update was more than 30 days ago, please do a full reset"
```

Comment:
Could you please move this into I18n? And perhaps reword to `please reset your Quickbooks data`.